File: APIManga/Model/Author.cs
﻿using System.ComponentModel.DataAnnotations;

namespace APIManga.Model
{
	public class Author
	{
		[Key]
		public int Id { get; set; }
		public string Name { get; set; }
		
		public ICollection<Manga> Mangas { get; set; }
	}
}

File: backend/app/models/author.py
from sqlalchemy import Column, Integer, String
from sqlalchemy.orm import relationship
from app.database.connection import Base

class Author(Base):
    __tablename__ = "author"

    id = Column(Integer, primary_key=True)
    name = Column(String, nullable=False)

    # RELAÇÕES
    mangas = relationship(
        "Manga",
        secondary="manga_author",
        back_populates="authors"
    )